packages/admin/cms-admin/src/blocks/table/ActionsCell.tsx

```diff
@@ -74,6 +83,94 @@ export const ActionsCell = ({ row, updateState }: Props) => {
         });
     };
 
+    const copyRowToClipboard = () => {
+        const rowToCopy = state.rows.find(({ id }) => id === row.id);
+        if (!rowToCopy) {
+            snackbarApi.showSnackbar(
+                <Snackbar autoHideDuration={5000}>
+                    <Alert severity="error">
+                        <FormattedMessage id="comet.tableBlock.failedToCopyRow" defaultMessage="Failed to copy row" />
+                    </Alert>
+                </Snackbar>,
+            );
+            return;
+        }
+
+        const copyData: ClipboardRow = {
+            highlighted: rowToCopy.highlighted,
+            cellValues: state.columns.map(({ id: columnId }) => {
+                const cellValue = rowToCopy.cellValues.find(({ columnId: cellColumnId }) => cellColumnId === columnId);
+                return cellValue ? cellValue.value : "";
+            }),
+        };
+
+        navigator.clipboard.writeText(JSON.stringify(copyData));
```

Comment:
We already have functions that handle clipboard write/read with permission check and fallback here: https://github.com/vivid-planet/comet/tree/main/packages/admin/admin/src/clipboard

Reply:
Done: b5846d058a338b9fd88bdbce45431694dfd3ea98